packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -136,38 +135,16 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
   }
 
   @autobind
-  private _onFocus(ev: React.FocusEvent<HTMLInputElement>): void {
-    const { inputProps } = this.props;
-
-    if (inputProps && inputProps.onFocus) {
-      inputProps.onFocus(ev);
-    }
-
-    this.setState({ isFocused: true });
-  }
-
-  @autobind
-  private _onBlur(ev: React.FocusEvent<HTMLInputElement>): void {
-    const { inputProps } = this.props;
-
-    if (inputProps && inputProps.onBlur) {
-      inputProps.onBlur(ev);
-    }
-
-    this.setState({ isFocused: false });
-  }
-
-  @autobind
-  private _onChange(ev: React.FormEvent<HTMLInputElement>) {
+  private _onClick(ev: React.FormEvent<HTMLButtonElement>) {
```

Comment:
You changed this to onClick, but the prop is still called onChange, they should align to keep it clear